tests/Generator/Instantiator/Chainable/StaticFactoryInstantiatorTest.php

```diff
@@ -244,7 +244,7 @@ public function testThrowsAnExceptionIfFixtureClassDoesNotMatchObjectClass()
 
     /**
      * @expectedException \Nelmio\Alice\Throwable\Exception\Generator\Instantiator\InstantiationException
-     * @expectedExceptionMessage Instantiated fixture was expected to be an instance of "Nelmio\Alice\Entity\Instantiator\DummyWithFakeNamedConstructor". Got "Nelmio\Alice\Throwable\Exception\Generator\Instantiator\InstantiationExceptionFactory" instead.
+     * @expectedExceptionMessage Instantiated fixture was expected to be an instance of "Nelmio\Alice\Entity\Instantiator\DummyWithFakeNamedConstructor". Got "null" instead.
```

Comment:
actually not, in PHP <7.2 `get_class(null)` would return the current class. Since PHP 7.2 it throws a `TypeError`